config.js

```diff
@@ -3,188 +3,23 @@ const templateData =
-              },
-              {
-                title: "click",
-                link: ""
-              },
-              {
-                title: "submit-click",
-                link: ""
-              },
-              {
-                title: "update",
-                link: ""
-              },
-              {
-                title: "timer",
-                link: ""
-              },
-              {
-                title: "redirect",
-                link: ""
-              },
-              {
-                title: "copy",
-                link: ""
-              },
-              {
-                title: "pause",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Functions",
-            link: "",
-            sub_items: [
-              {
-                title: "clear cookies",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Play",
-            link: ""
-          },
-          {
-            title: "Clipboard",
-            link: ""
-          },
-          {
-            title: "Options",
-            link: "",
-            sub_items: [
-              {
-                title: "CBA",
-                link: ""
-              },
-              {
-                title: "Functions Option",
-                link: ""
-              },
-              {
-                title: "Import-Export",
-                link: ""
-              }
-            ]
-          }
-        ]
-      },
-      {
-        title: "Tutorial",
-        link: "#",
-        sub_items: [
-          {
-            title: "Registration Demo",
-            link: ""
-          },
-          {
-            title: "Facebook Demo",
-            link: ""
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Video Presentation for CBA",
-            link: ""
-          }
-        ]
-      },
-      {
-        title: "Ready Projects",
-        link: "#",
-        sub_items: [
-          {
-            title: "FB like/unlike",
-            link: ""
-          }
-        ]
-      },
-      {
-        title: "What's New",
-        link: "#",
-        sub_items: [
-          {
-            title: "Version 8.0",
-            link: ""
-          },
-          {
-            title: "Version 6.0",
-            link: ""
-          },
-          {
-            title: "Version 5.0",
-            link: ""
-          }
-        ]
+    menuItems: ["", "documentation", "tutorial", "ready-projects",
+      "what's-new"],
+    pagesList: function (absolutePath) {
+      optionsList = templateData.site.queryPages(({originalPathname}) => {
+        return originalPathname && originalPathname.split('/')[0] == absolutePath.split('/')[0] && 
+            originalPathname.split('/')[1] == absolutePath.split('/')[1]});
```

Comment:
Seems like this logic doesn't work only for direct subpages of the page, but also with subpages of subpages.

```
// Consider we have originalPathname of the current page
query((page) => page.originalPathname.startsWith(originalPathname) &&
  page.originalPathname.replace(originalPathname, "").split("/").length === 2);
```

Reply:
> Seems like this logic doesn't work only for direct subpages of the page, but also with subpages of subpages.

don't really get what's the issue... :(

Reply:
Your goal here is to list direct children of `absolutePath` passed above, so for example if you have:
```
/documentation/actions/inject
/documentation/actions/change
/documentation/actions/change/more-change
```

And pass `/documentation/actions` as `absolutePath`, your script will list all the items, while I assume the expected result is to list only:
```
/documentation/actions/inject
/documentation/actions/change
```

There is a fix for that in the comment also.

Reply:
Then if we do like in this -> https://github.com/browser-automation/cba-website/pull/18#discussion_r429625793 comment I think the issue will be resolved.